### PR TITLE
update disclaimer on audits

### PR DESCRIPTION
**Pull request by marcialc**
Screen Shot 2024-03-12 at 10 29 27 AM

### CODE REVIEW

File: packages/docsite/src/pages/docs/index.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ Modular Governance is a set of smart contracts that can be used to create a gove
 ---
 
 {% callout type="warning" title="DISCLAIMER" %}
-It is important to know that Helium Modular Governance is still in Alpha stages and is subject to change. We are working hard to make sure that the contracts are secure and that the API is stable. If you have any questions or concerns please reach out to us on [Discord](https://discord.gg/3Xk3F2Q6).
+It's essential to be aware that Helium Modular Governance is currently in its Alpha phase and may undergo modifications. Except for the NFT voter program, all contracts have been audited. Our team is dedicated to ensuring the security of the contracts and the stability of the API. Should you have any inquiries or concerns, please do not hesitate to contact us on [Discord](https://discord.gg/3Xk3F2Q6).
```
Comment (abhay) [inline packages/docsite/src/pages/docs/index.md:11]:
```suggestion
It's essential to be aware that Helium Modular Governance is currently in its Alpha phase and may undergo modifications. Except for the NFT voter program, all contracts have been audited. Our team is dedicated to ensuring the security of the contracts and the stability of the API. Should you have any inquiries or concerns, please do not hesitate to contact us on [Discord](https://discord.gg/helium).
```

Comment (abhay) [reply packages/docsite/src/pages/docs/index.md:11]: small nit. also @ChewingGlass do you want to say it's no longer Alpha? we're in production with it :)

Comment (ChewingGlass) [reply packages/docsite/src/pages/docs/index.md]: I'm good with that.

Comment (ChewingGlass) [reply packages/docsite/src/pages/docs/index.md]: Beta until Solana is out of beta? lol

Comment (marcialc) [reply packages/docsite/src/pages/docs/index.md]: @abhay @ChewingGlass Should I just call it beta?
that sounds good to me

Comment (abhay) [reply packages/docsite/src/pages/docs/index.md]: yep. makes sense

Comment (marcialc) [reply packages/docsite/src/pages/docs/index.md]: updated

**Review by abhay — APPROVED** [review]

**Pull request merged**